### ISSUE TITLE
Proposal Workflow Editor Accessibility Issues

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1655]
The following are impacting accessibility for users trying to use the proposal workflow editor:
- Workflow properties fields (title, description) should have labels
- Status events and actions modal - next status events checkboxes label isn't read out by the NVDA screen reader when the field is focused
- Editing workflows isn't possible with just keyboard input (https://reactflow.dev/learn/advanced-use/accessibility)
- "Connection line style" text has low contrast
- The list of available statuses has children that are not allowed elements with `role=button`
